ajax/libs/hyperHtml/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "hyperHtml",
+  "filename": "min.min.js",
```

Comment:
We can just use `min.js` because it's already minified :slightly_smiling_face:

Reply:
see https://github.com/cdnjs/cdnjs/pull/12547#pullrequestreview-99160494